### ISSUE TITLE
update 

### ISSUE THREAD

**Issue by arderyp** [symfony/webpack-encore#1162]
related issue: https://github.com/symfony/webpack-encore/issues/1144

I'm running `@symfony/webpack-encore@4.1.1`

```
$ yarn upgrade
yarn upgrade v1.22.19
warning @symfony/webpack-encore > css-minimizer-webpack-plugin > cssnano > cssnano-preset-default > postcss-svgo > svgo > rn JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
```

```
$ yarn why stable
yarn why v1.22.19
=> Found "stable@0.1.8"
info Reasons this module exists
   - "@symfony#webpack-encore#css-minimizer-webpack-plugin#cssnano#cssnano-preset-default#postcss-svgo#svgo" depends on it
   - Hoisted from "@symfony#webpack-encore#css-minimizer-webpack-plugin#cssnano#cssnano-preset-default#postcss-svgo#svgo#stable"
```

This issue has been fixed downstream: https://github.com/svg/svgo/pull/1681

Can we somehow resolve this now? Or do we need to wait on one of our dependencies to update?

**Comment by stof** [comment]
This has been resolved in svgo 3.0. So the next step is that `postcss-svgo` needs to update to the new major version of svgo. this is in progress at https://github.com/cssnano/cssnano/pull/1440, but will need to be part of a new major version of cssnano as well, which will take a bit of time.
There is nothing webpack-encore can do to remove that warning except waiting for the update to propagate in the dependency chain.

**Comment by arderyp** [comment]
Thanks @stof 
Is https://github.com/cssnano/cssnano/pull/1440 the only remaining issue for the fullpropogration?

if not, what other issues should I be subscribing to>

**Comment by stof** [comment]
As they will release that as a major version, I'm quite sure you will need another issue then on `css-minimizer-webpack-plugin` to update to cssnano 6. But this issue does not exist yet as cssnano does not exist yet. And then maybe a ticket in webpack-encore to update to a new major of `css-minimizer-webpack-plugin` depending on whether the support for cssnano 6 comes into a major or minor version (we would need it only if they do it in a major version).

**Comment by arderyp** [comment]
Thanks for the clarification @stof.

Do you think it's okay to leave this issue open until the fix percolates upward and consumers of this package no longer experience this issue? Or would you rather I close it?

**Comment by iquito** [comment]
As far as I can tell all preceding steps in both `cssnano` and `css-minimizer-webpack-plugin` have now happened, webpack-encore just needs to update its dependency of `css-minimizer-webpack-plugin` from `^4.0.0` to `'^5.0.0`.

**Comment by arderyp** [comment]
hey @iquito @weaverryan @fabpot can we make the change suggested in the comment above?

**Comment by stof** [comment]
Closing this as the upgrade has been done in 4.4.0: https://github.com/symfony/webpack-encore/releases/tag/v4.4.0
